src/pages/BooksDetail.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from "react";
+import { motion } from "framer-motion";
+import AnimateButton from "../components/AnimateButton";
+import coverImg from "../assets/cover.jpg";
+import page1 from "../assets/page1.png";
+import page2 from "../assets/page2.png";
+import page3 from "../assets/page3.png";
+import Zoom from "react-medium-image-zoom";
+import "react-medium-image-zoom/dist/styles.css";
+
+function BooksDetail() {
```

Comment:
How are we getting the data of the book being displayed? Are we passing Id or getting id from the router? Can you handle that here! 
If router (preferable) then just use the route and get the id, and use show how will you fetch the data.

Reply:
PS: I made changes to the routing and displaying books in this [PR](https://github.com/Iraq-RBC-Capstones-2021/capstone-react-libvery/pull/28).